bundler/spec/bundler/definition_spec.rb

```diff
@@ -56,6 +56,11 @@
         s.add_dependency "rack", "1.0"
       end
 
+      checksums = checksums_section_when_existing do |c|
+        c.no_checksum "foo", "1.0"
+        c.checksum gem_repo1, "rack", "1.0.0"
+      end
+
```

Comment:
This should go _after_ the `bundle install` command below. Otherwise it will generate an empty section once we generate checksums by default and will make the spec fail.